### PR TITLE
Add customize-able project labels applying to all fuzz targets in a Project

### PR DESCRIPTION
**Pull request by asraa**
Signed-off-by: Asra Ali <asraa@google.com>

Part of https://github.com/google/oss-fuzz/issues/7318

In Project setup, checks for a wildcard label defined in the `project.yaml`'s `labels` field to apply project-wide labels when filing issues. (Previously only per-fuzz target labels were supported). See issue for use-case.

### CODE REVIEW

**Comment by asraa** [comment]
cc @krajshiva

File: src/appengine/handlers/cron/project_setup.py [hunk 1]
```diff
@@ -790,11 +790,14 @@ def _sync_job(self, project, info, corpus_bucket_name, quarantine_bucket_name,
...
-                engine=template.engine,
-            ))
+        automatic_labels = 'Proj-{project},Engine-{engine}'.format(
+            project=project,
+            engine=template.engine,
+        )
+        labels = info.get('labels')
+        if labels and '*' in labels:
+          automatic_labels += ',%s' % ','.join(labels['*'])
+        job.environment_string += ('AUTOMATIC_LABELS = %s\n' % automatic_labels)
```
Comment (oliverchang) [inline src/appengine/handlers/cron/project_setup.py:800]: nit: can we clean this up a bit by doing:

```python
automatic_labels = [
  f 'Project-{project}',
  f 'Engine-{engine}',
]

labels = info.get('labels')
if labels and '*' in labels:
  automatic_labels.extend(labels['*'])

automatic_labels = ','.join(automatic_labels)
job.environment_string += f'AUTOMATIC_LABELS = {automatic_labels}\n'
```
```

**Review by oliverchang — APPROVED** [review]
Thanks a lot for implementing this @asraa !!!

LGTM -- Just one comment on making this a bit cleaner.

File: src/appengine/handlers/cron/project_setup.py (hunk 1 above)
Comment (asraa) [reply src/appengine/handlers/cron/project_setup.py]: nice, done! thanks

**Pull request merged**